test/VariablesTest.cs

```diff
@@ -11,15 +11,24 @@ public VariablesTest()
         }
 
         [Test]
-        public void TestValue()
+        public void TestGetValue()
         {
             Mocker.Mock(new Uri("test:///variables/foo%2Fbar/value"), "testvalue");
             Assert.AreEqual("testvalue", Client.Variable("foo/bar").GetValue());
 
             // TODO: not sure if this is supposed to be a plus or %20 or either
-            Mocker.Mock(new Uri("test:///variables/foo+bar/value"), "space test");
+            Mocker.Mock(new Uri("test:///variables/foo%20bar/value"), "space test");
             Assert.AreEqual("space test", Client.Variable("foo bar").GetValue());
         }
+
+        [Test]
+        public void TestAddValue()
+        {
+            Mocker.Mock (new Uri ("test:///variables/foo%2Fbar/values"), "testvalue");
+            Client.Variable ("foo/bar").AddValue ("testvalue");
+
+            string res = Mocker.Mock (new Uri ("test:///variables/foo%2Fbar/value"), "testvalue").Read ();
+            Assert.AreEqual (res, "testvalue");
```

Comment:
What's actually happening here? Are we just creating a mock object to return a set string and verifying the string matches? If so, it feels like we remove any real testing value by mocking out the response.

Reply:
I agree, this test does not appear to actually test anything.

Reply:
Note how it still passes if you remove `AddValue` implementation.